pkg/agent/config_test.go

```diff
@@ -97,8 +98,8 @@ func Test_ValidateAndCombineConfig(t *testing.T) {
 			withCmdLineFlags("--period", "99m", "--credentials-file", fakeCredsPath))
 		require.NoError(t, err)
 		assert.Equal(t, testutil.Undent(`
-			Using the Jetstack Secure OAuth auth mode since --credentials-file was specified without --venafi-cloud.
-			Both the 'period' field and --period are set. Using the value provided with --period.
+			INFO Using the Jetstack Secure OAuth auth mode since --credentials-file was specified without --venafi-cloud.
+			INFO Both the 'period' field and --period are set. Using the value provided with --period.
```

Comment:
Why is this output not matching the klog text format and uses a custom format?

Reply:
Because these tests are using the ktesting test logger which has a buffer containing simplified output format for easier testing and matching. 
The log format of the `preflight`  command is tested elsewhere.
I'll leave this as-is.